src/components/mainContent/DirectionButtons.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./mainContent.css";
+import arrows from "../../assets/upArrow.svg";
+import content from "../../assets/labContent.js";
+export default function DirectionButtons(props) {
+  function handleBackward() {
+    let previousSection = content[props.visibleSections - 1];
+    let previousLength = undefined;
+    if (previousSection && previousSection.post)
+      previousLength = previousSection.post.bodyText.length;
+    else if (previousSection && previousSection.followups)
+      previousLength = previousSection.followups.length;
```

Comment:
This can be written more succinctly with a ternary operator; but, to check, should there ever be a situation where `previousLength = undefined` after this block finishes? That may not be a great design choice, since you never check `previousLength` afterwards

Reply:
Yeah, setting it up in a ternary makes more sense, and there's no way that it'll end up undefined anyways (since forward button only cares about current section, not previous section)